backend/fms_core/template_importer/importers/axiom_preparation.py

```diff
@@ -62,7 +71,9 @@ def import_template_inner(self):
                 if i < self.properties_starting_index:
                     sample_preparation_dict[key] = row[key]
                 else:
-                    process_properties[key]['value'] = val
+                    # Convert problematic header names
+                    combined_key = TEMPLATE_PROPERTY_MAPPING[key] if key in TEMPLATE_PROPERTY_MAPPING.keys() else key
```

Comment:
Shouldn't `in` be checking the presence of a key already for python dictionaries?

> 
> key in d
> 
>     Return True if d has a key key, else False.
>

Reply:
i prefer stating the not so obvious rather than forcing someone to check on type and behaviour while reading the code ...